product_pricelist_supplierinfo_coefficient/views/product_pricelist_item_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="product_pricelist_item_form_view_inherit" model="ir.ui.view">
```

Comment:
Adding this field in product form view (in Purchase Prices) is missing, see requirements

Reply:
Still missing this

Reply:
Added the field in 2 views, and I have not had problems with model.